{"name": "Ошибка авторизации при отсутствии", "status": "passed", "description": "Проверка, что авторизация не осуществляется из-за отсутствующих данных", "parameters": [{"name": "payload", "value": "{'login': '', 'password': '1234'}"}], "start": 1701558796321, "stop": 1701558796521, "uuid": "590177e1-3324-44a5-ad64-8bfcaeaa3310", "historyId": "81ca1fc42b2d18df5d697d2e43fbaddd", "testCaseId": "dfe2ffe8def4ccd29525bbdb5297be30", "fullName": "test.test_login_courier.TestLoginCourier#test_login_courier_empty_field_error_not_enough_data", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-QRA353I"}, {"name": "thread", "value": "13296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_login_courier"}]}